mkt/constants/base.py

```diff
@@ -92,6 +93,18 @@
     'unlisted': STATUS_UNLISTED,
 }
 
+STATUS_FILE_CHOICES_API_v2 = {
```

Comment:
`STATUS_FILE_CHOICES` had the whole list so I also used the whole list here, even though some might be useless. I prefer to not limit this right now, I don't really want to dive deep into the code to find out which ones are not useful right now.